src/components/Editor/styles.css

```diff
@@ -55,4 +55,3 @@
 .contentWrapper {
     display: contents;
 }
```

Comment:
restore empty line at the end of the file

Reply:
there is one, it was doubled :)

Reply:
cool